Resources/Prototypes/Traits/skills.yml

```diff
@@ -168,3 +168,19 @@
   components:
     - type: Singer
       proto: NormalSinger
+
+- type: trait
+  id: LatentPsychic
+  category: Mental
+  points: -3
+  components:
+    - type: Psionic
+  requirements:
+    - !type:CharacterJobRequirement
+      inverted: true
+      jobs:
+        - Borg
+        - MedicalBorg
+        - ResearchDirector
+        - ForensicMantis
+        - Chaplain
```

Comment:
Why can't the Chaplain take this?

They're not given a power

Reply:
I'm assuming either the Chaplain is Psionic by default (and therefore doesn't need the Latent trait), or is given Latent by his job.

Reply:
The Chaplain instead has a Blank PsionicComponent now, meaning he is automatically the new equivalent of a "Latent Psychic". Which is going to be necessary for when I do Golemancy as its own PR.